go/src/koding/klient/machine/index/node_test.go

```diff
@@ -211,19 +145,19 @@ func TestNodeLookup(t *testing.T) {
 				t.Fatalf("Lookup(%q) failed", name)
 			}
 
-			if nd.Entry.Size != size {
-				t.Fatalf("got %d, want %d", size, nd.Entry.Size)
+			if nd.Entry.Size() != size {
+				t.Fatalf("got %d, want %d", size, nd.Entry.Size())
 			}
 		})
 	}
 }
 
 func TestNodeCount(t *testing.T) {
 	cases := map[int64]int{
-		-1:   32,
+		-1:   33,
```

Comment:
> Since now, Index will store root Entry during initialization. Its name/path will be empty string "" and not "." since we do not use dots in other entries.